tests/details/loongarch.yaml

```diff
@@ -120,3 +120,17 @@ test_cases:
                 type: LOONGARCH_OP_IMM
                 imm: 0x0
           groups: [ LOONGARCH_GRP_JUMP ]
+      -
+        asm_text: "ld.bu	$t0, $t4, -1"
+        details:
+          loongarch:
+            operands:
+              -
+                type: LOONGARCH_OP_REG
+                reg: t0
+              -
+                type: LOONGARCH_OP_MEM
+                mem_base: t4
+                mem_disp: -1
+          regs_read: [ t4 ]
+          regs_write: [ t0 ]
```

Comment:
Please also add `access: CS_AC_WRITE` and `access: CS_AC_READ` to the operands of this test case.

Reply:
Done.